parsers/s01-parse/firix/authentik-logs.yaml

```diff
@@ -0,0 +1,24 @@
+name: firix/authentik-logs
+description: "Parse Authentik logs"
+filter: "Lower(evt.Parsed.program) == 'authentik'"
+onsuccess: next_stage
+nodes:
+  - filter: "JsonExtract(evt.Line.Raw, 'action') == 'login_failed'"
```

Comment:
Could we use `evt.Parsed.message` instead of raw? sometimes, user want to log to another engine like syslog file this would cause an issue.